web/www/horas/English/Sancti/09-12.txt

```diff
@@ -16,7 +16,7 @@ Doxology=Nat
 BMV
 
 [Ant 1]
-O Holy Mary, be thou an help to the helpless, a strength to the fearful, a~
+O Holy Mary, * be thou an help to the helpless, a strength to the fearful, a~
```

Comment:
I'm happy for you to move the star to match the Latin where possible.  Would you prefer me to merge the change as it is now, or would you rather move those stars?